### PR TITLE
Fix Python Athena connection instructions

### PR DESCRIPTION
**Pull request by dfsnow**
Quick fixes for the Python Athena directions in the wiki, as they were incomplete.

### CODE REVIEW

File: How-To/Connect-to-AWS-Resources.md [hunk 1]
```diff
@@ -105,22 +105,22 @@ As with R, enabling [unload](https://laughingman7743.github.io/PyAthena/pandas.h
     # Load necessary libraries
     import os
     import pandas
+    import pyarrow
```
Comment (dfsnow) [inline How-To/Connect-to-AWS-Resources.md:108]: This is a required dependency for using unload.

File: How-To/Connect-to-AWS-Resources.md [hunk 2]
```diff
@@ -105,22 +105,22 @@ As with R, enabling [unload](https://laughingman7743.github.io/PyAthena/pandas.h
...
-        # when the option is disabled.
+        # requires that the staging directory end with a slash
         s3_staging_dir=os.getenv("AWS_ATHENA_S3_STAGING_DIR") + "/",
         region_name=os.getenv("AWS_REGION"),
         cursor_class=PandasCursor,
     ).cursor(unload=True)
 
-    # Define test query
-    SQL_QUERY = "SELECT * from default.vw_pin_sale LIMIT 10;"
+    # Define test query. Note that the query CANNOT end with a semi-colon
```
Comment (dfsnow) [inline How-To/Connect-to-AWS-Resources.md:122]: This also causes errors when trying to unload, so good to have a note about it here.

File: How-To/Connect-to-AWS-Resources.md [hunk 3]
```diff
@@ -105,35 +105,35 @@ As with R, enabling [unload](https://laughingman7743.github.io/PyAthena/pandas.h
...
+    SQL_QUERY = "SELECT * from default.vw_pin_sale LIMIT 10"
 
     # Execute query and return as pandas df
     cursor.execute(SQL_QUERY)
 
-    df = as_pandas(cursor)
+    df = cursor.as_pandas()
```
Comment (dfsnow) [inline How-To/Connect-to-AWS-Resources.md:128]: We're already using the `PandasCursor` class, so using the built-in method actually makes this slightly faster.

**Review by wrridgeway — APPROVED** [review]

File: How-To/Connect-to-AWS-Resources.md (hunk 3 above)
Comment (wrridgeway) [reply How-To/Connect-to-AWS-Resources.md:128]: This is great news.

**Pull request merged**